src/components/BaseComponents/Modal/Modal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Modal(props) {
+  const { handleClose, show, children } = props;
+  const showHideClassName = show
+    ? 'govuk-!-display-block hmrc-timeout-dialog'
+    : 'govuk-!-display-none';
+
+  return (
+    show && (
+      <>
+        <div className='hmrc-timeout-overlay'></div>
+        <div className={showHideClassName} tabIndex={-1} role="dialog" aria-modal="true" >
+          <section>
+            {' '}
```

Comment:
Why ??

Reply:
I think that has been added automatically while formating the document with prettier